src/main.rs

```diff
@@ -33,7 +33,7 @@ impl Person {
 
 fn main() -> io::Result<()> {
     let mut bdays = Bdays::new();
-    let path: &String = &format!("/home/{}/.config/bdays.json", env!("USER"));
+    let path: &String = &format!("{}/.config/bdays.json", env!("HOME"));
```

Comment:
/home/user only works on linux

Reply:
Funny cuz the change you made is also linux only if I'm not mistaken

Reply:
I see, ty!

Reply:
> Funny cuz the change you made is also linux only if I'm not mistaken

wait yeah, / doesn't work on windows paths right?

Reply:
eh doesn't really matter, this is targeted towards linux users anyway :rofl: